bento/src/main/java/com/yelp/android/bento/componentcontrollers/RecyclerViewComponentController.java

```diff
@@ -59,13 +60,11 @@ public RecyclerViewComponentController(@NonNull RecyclerView recyclerView) {
     /**
      * Creates a new {@link RecyclerViewComponentController} and automatically attaches itself to
      * the {@link RecyclerView}. In order to make the lanes (columns / rows), this component
-     * controller will set the {@link RecyclerView}'s {@link RecyclerView.LayoutManager}. Do not do 
+     * controller will set the {@link RecyclerView}'s {@link RecyclerView.LayoutManager}. Do not do
      * it manually.
      */
     public RecyclerViewComponentController(
-            @NonNull RecyclerView recyclerView,
-            @Orientation int orientation
-    ) {
```

Comment:
Just curious, are you using a kotlin formatter? Is it something we should consider adding to bento?

Reply:
No, but I am using a Java formatter: https://github.com/google/google-java-format